src/tasks/DockerBuildTaskProvider.ts

```diff
@@ -7,14 +7,12 @@ import * as fse from 'fs-extra';
 import * as path from 'path';
 import { l10n, Task } from 'vscode';
 import { DockerPlatform } from '../debugging/DockerPlatformHelper';
-import { ext } from '../extensionVariables';
 import { cloneObject } from '../utils/cloneObject';
 import { resolveVariables } from '../utils/resolveVariables';
 import { DockerBuildOptions } from './DockerBuildTaskDefinitionBase';
 import { DockerTaskProvider } from './DockerTaskProvider';
 import { NetCoreBuildTaskDefinition } from './netcore/NetCoreTaskHelper';
 import { NodeBuildTaskDefinition } from './node/NodeTaskHelper';
-import { defaultVsCodeLabels, getAggregateLabels } from './TaskDefinitionBase';
 import { DockerBuildTaskContext, TaskHelper, throwIfCancellationRequested } from './TaskHelper';
 
 export interface DockerBuildTaskDefinition extends NetCoreBuildTaskDefinition, NodeBuildTaskDefinition {

```

Comment:
We should revisit the build/run task inheritance, as the current task definition inheritance is the opposite of the TaskHelper inheritance. Theoretically we should be able to structure the types such that we won't need to explicitly cast between types when calling the base methods on TaskHelper.